shared/ios/KeybaseShare/ShareViewController.m

```diff
@@ -106,280 +99,14 @@ - (void) openApp {
   }
 }
 
-- (void) startProcessingAlreadyInMainThread:(NSInteger)items {
-  self.unprocessed = items;
-}
-
 - (void) completeRequestAlreadyInMainThread {
-  [self writeManifest];
   [self.alert dismissViewControllerAnimated:true completion:^{
     [self.extensionContext completeRequestReturningItems:nil completionHandler:^(BOOL expired) {
       [self openApp];
     }];
   }];
 }
 
-- (void) completeProcessingItemAlreadyInMainThread {
```

Comment:
we pull this into a helper since we want a lot of this logic in our new drop handling